### PR TITLE
[benchmark] Fixes #206  Add SSD MobileNet Model to Benchmark test.

### LINKED ISSUE #206: [benchmark] Add SSD Mobilenet into Benchmark test.

### PR DESCRIPTION
**Pull request by BruceDai**
@huningxin PTAL,thanks

### CODE REVIEW

File: examples/ssd_mobilenet/utils.js [hunk 1]
```diff
@@ -139,316 +135,4 @@ class Utils {
       }
     }
   }
-
-  /**
-  * Decode out box coordinate
```
Comment (huningxin) [inline examples/ssd_mobilenet/utils.js]: why move those functions to base.js. I suppose those are only used by SSD example. Correct?

Comment (BruceDai) [reply examples/ssd_mobilenet/utils.js]: I will reuse these functions for benchmark test with SSD MobileNet, so move them to base.js.thanks

**Comment by huningxin** [comment]
@ibelem , please help review the PR thanks!

**Comment by ibelem** [comment]
LGTM as it's been applied in AIMark. There will be demo/benchmark sharing code and UI design review to improve some codes.

**Pull request merged**